docker/README.md

```diff
@@ -57,3 +58,28 @@ Example to build a custom sink service tagged local/my_sink_service:tag1 in your
 docker build -f docker/sink-service/Dockerfile -t local/my_sink_service:tag1
 ```
 You can then use this image in the docker-compose template instead of the Wirepas Docker Hub ones.
+
+## Setup under WSL2
+The Wirepas gateway driver can be deployed under Windows Subsystem for Linux 2 aka WSL2. Its setup procedure is described in this section for the Ubuntu 22.04LTS distribution. It should work with any other but can require some adaptation to what is provided.  
+
+### Prerequisites
+* Windows 10 or 11 (Build 22000 or later) installed
+
+### Flow
+Install (or update) WSL v2 package. Its version must be \>= v1.0  
```

Comment:
is \ before >= on purpose? Escape char for md ?

Reply:
It is on purpose and as you said an escape sequence